client/src/components/EditForm/index.tsx

```diff
@@ -107,14 +107,13 @@ export const EditForm: FC<EditFormProps> = ({
       clearInput?.();
     } else {
       const updatedTodo = {
-        _id,
         text,
         time: {
           start,
           end,
         },
       };
-      dispatch(editTodo(updatedTodo));
+      dispatch(editTodo({ id: _id, todo: updatedTodo }));
```

Comment:
You can pass to editTodo todo with id field and inside editTodo get id from object